composer.json

```diff
@@ -257,9 +263,17 @@
         },
         "runtime": {
             "dotenv_path": "project-base/app/.env"
+        },
+        "enable-patching": true,
+        "patches": {
+            "composer-exit-on-patch-failure": true,
+            "twig/twig": {
+                "Hotfix for issue https://github.com/sonata-project/SonataAdminBundle/issues/8181": "https://github.com/TomasLudvik/Twig/commit/4e07748eb6d427688221e2c7b6dc8872c6d9065c.patch"
```

Comment:
It needs to be provided via URL (don't ask me why) and I did not find a way to load it from the local filesystem. It could be pushed to shopsys/shopsys and then linked, but maintaining fork and just changing those few line seemed to be much easier. 

Do you want me to push it to shopsys/shopsys and then link on it?

Reply:
according to https://github.com/cweagans/composer-patches/tree/1.7.3#using-an-external-patch-file the local fila should be supported

Reply:
Ok, I found a way, it is implemented now